canonical-web-and-design/ubuntu.com
# Update sitemap.xml for static files

## Done

- Update sitemap.xml for static files
- I changed my script to just look at the files in the /templates directory as we now have code to generate most of the module created pages (docs, engage pages, tutorials, cves, usns).

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/static/files/sitemap.xml
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that it looks correct and validates
